roles/artemis/defaults/main.yml

```
@@ -105,6 +105,8 @@ artemis_version_control_default_branch_name: main
 
 enable_science_event_logging: false
 
+artemis_theia_base_url: https://theia-yannik.k8s.ase.cit.tum.de
+
```

Are you planning to add more configuration options in the future? If so, I suggest adding an additional key for theia related configuration:

```suggestion
theia:
  base_url: https://theia-yannik.k8s.ase.cit.tum.de
```

Please also comment out the configuration in the defaults. We use the defaults as a reference for possible configuration options for external systems.

---

Thanks, did it exactly your way 💯